src/components/Header/PipelineLog.js

```diff
@@ -57,7 +71,8 @@ const Pipeline = ({pipelineKey: key, data, socket}) => {
           <span className="rightIcon">
             <IoIosAlert className="icon150" color="#F6EECA" />
           </span>
-        ) : null }
+        ) : null 
```

Comment:
did we need to change the formatting? This introduces extra diff which isn't needed

Reply:
no, we didn't